webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -388,13 +535,14 @@ def update_info_text(
         ):
             return []
         info_text = []
-        ensemble = load_ensemble(parent, ensembles[0])
         active_response = active_resp_param["response"]
-        x_axis = ensemble.responses[active_response].axis
-        formatted_active_index = _format_index_value(
-            x_axis, corr_xindex.get(active_response, 0)
-        )
-        active_index_text = str(formatted_active_index)
+
+        selected_obs_indexes = selected_obs.get(active_response, None)
```

Comment:
this variable name should be singular, right? it's just one index?
and perhaps renamed to active_observation_index?